From 735bb602823ea79d8657889a195573fcc053c143 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Wed, 22 Sep 2021 14:46:11 +0300
Subject: [PATCH] [SW-0] gaudi2: arc: fix white space errors

No functional changes - only a couple of spaces changed to tabs.

Change-Id: I42da6642e88e5e33b979a8ef128ed07898dadc05
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/arch/gaudi2/arc/common.h b/arch/gaudi2/arc/common.h
index 023b35a..9134b2c 100644
--- a/arch/gaudi2/arc/common.h
+++ b/arch/gaudi2/arc/common.h
@@ -19,8 +19,8 @@
 #define HL_SP_SRAM_SIZE		0x10000
 
 #ifdef TARGET_IS_ONIX
-#define SDRAM_BASE              0x10000000
-#define SDRAM_FULL_BASE         0x1001000200000000ull
+#define SDRAM_BASE		0x10000000
+#define SDRAM_FULL_BASE		0x1001000200000000ull
 #define ARM_ADDR_MASK		0xFFFFFFFull
 #else
 /*
@@ -29,8 +29,8 @@
  * SRAM array starts at offset of 16MB from the region start and is of 48MB size
  * That makes the SRAM reside in range of 0x11000000 - 0x13FFFFFF (inclusive).
  */
-#define SDRAM_BASE              0x11000000
-#define SDRAM_FULL_BASE         0x1000FFFFFD000000ull
+#define SDRAM_BASE		0x11000000
+#define SDRAM_FULL_BASE		0x1000FFFFFD000000ull
 #define ARM_ADDR_MASK		0x1FFFFFFull
 #endif
 
